doc/templates/uno/styles/main.css

```diff
@@ -3,6 +3,10 @@
+@media(min-width: 1024px){.visibility-opacity-transition,#header-container nav>ul>li.menu-item-has-children>ul ul,#header-container nav>ul>li.menu-item-has-children>ul{visibility:hidden;opacity:0;transition:visibility 100ms,opacity 100ms ease-in-out;list-style:none}}#header-container{padding:16px 32px;z-index:9999;background:#fff;display:flex;position:fixed;width:100%;min-height:67px;max-height:67px;border-bottom:1px solid #f4f2f3;justify-content:space-between}#header-container::after,#header-container::before{content:none}#header-container #menu-menu-principal{float:right;list-style:none}@media(max-width: 1023px){#header-container #menu-menu-principal{width:100%;max-height:70vh;overflow-y:auto}}@media(min-width: 1024px){#header-container #menu-menu-principal{display:flex}}#header-container #header-logo-container{width:36px;vertical-align:middle;z-index:9}#header-container #header-logo-container img{width:100%}@media(min-width: 1024px){#header-container #header-logo-container{min-width:120px}}#header-container .navbar-toggle{padding:0;margin:10px 0;display:none !important;z-index:9}#header-container .navbar-toggle .icon-bar{background:#333;width:20px}#header-container .navbar-toggle .icon-bar+.icon-bar{margin-top:3px}@media(max-width: 1023px){#header-container .navbar-toggle{display:block !important}}#header-container .navbar-brand{height:auto}#header-container nav a{padding:10px;font-size:13px;font-weight:600;font-stretch:normal;font-style:normal;line-height:14px;letter-spacing:normal;text-transform:uppercase;color:#242424}#header-container nav a:hover,#header-container nav a:focus{color:#7a67f8}
```

Comment:
@marcoscodnet when I test these changes with the [related static link](https://github.com/unoplatform/uno/pull/19551#issuecomment-2666476871) related to the PR - https://unodocsprstaging.z13.web.core.windows.net/pr-19551/articles/intro.html, I am still able to see the issue happening when resizing the window and for narrow/phone breakpoint there is no "sign in" button at all:
- ![image](https://github.com/user-attachments/assets/a12b2dd4-a892-4a4f-861a-c431c6bd4e0b)
- ![image](https://github.com/user-attachments/assets/1728f5d0-203d-4500-b2b0-016fa9d463e1)
- ![image](https://github.com/user-attachments/assets/9f8dda8c-b437-4ea5-856e-9f620f1d0eac)